src/Function/Bundles.agda

```diff
@@ -189,6 +189,51 @@ module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
       }
 
 
+  record BiEquivalence : Set (a ⊔ b ⊔ ℓ₁ ⊔ ℓ₂) where
```

Comment:
Hmm is this definition needed or not? If it is then we should make sure we add a `IsBiEquivalence` record as well and make sure it's added to the CHANGELOG.

Reply:
OK, but `IsEquivalence` is is not a record itself, but defined as a field in the `Setoid` record.

Should we better add a `BiSetoid` record to `Relation/Binary/Bundles.agda` that includes a `IsBiEquivalence` field and the arrow symbol `_↩↪_` that you proposed in [574466050](https://github.com/agda/agda-stdlib/issues/1055#issuecomment-574466050)?

Reply:
> OK, but IsEquivalence is not a record itself, but defined as a field in the Setoid record.

I'm not quite sure what you mean by this. `IsEquivalence` is a record:
https://github.com/agda/agda-stdlib/blob/15dc42e1eeb0dbf946961213ffd86b5fa60ccb00/src/Relation/Binary/Structures.agda#L47

> Should we better add a BiSetoid record to Relation/Binary/Bundles.agda that includes a IsBiEquivalence field and the arrow symbol _↩↪_ that you proposed in 574466050?

No I don't think so. A `BiEquivalence` is still talking about functions between two types and therefore belongs in the function hierarchy. The `_↩↪_` symbol I was referring to, should be used for the instantiation of `BiEquivalence` with propositional equality. See the examples at the bottom of `Function.Bundles`.

Reply:
> I'm not quite sure what you mean by this. `IsEquivalence` is a record:

Sorry, when retrieving the `agda-mode:go-to-definition` command, I mixed `isEquivalence` with `IsEquivalence`. Of course, you are right, `IsEquivalence` is a record in `Relation/Binary/Structures.agda`

Thank you for the hint, I added a new BiInverse bundle `_↩↪_` and a new constructor `mk↩↪`  following the notation of the examples at the end of `Funtion.Bundles`